gulpfile.js

```diff
@@ -1380,6 +1380,7 @@ gulp.task("types", function (done) {
     "esModuleInterop",
     "forceConsistentCasingInFileNames",
     "emitDeclarationOnly",
+    "skipLibCheck",
```

Comment:
Please note that I've got no idea why adding this "just works", and whether or not it'll impact the generated types negatively, but some quick searching suggested using this and it worked :-)

(This kind of problem, with a cryptic error thrown by `tsc`, is what I was afraid of when support for generation of types was added. At least in *this* case, a simple work-around was at least available.)

Reply:
Alternatively, I think we can add ` --moduleResolution node`